test-platform-ui/src/app/administration/assessments/[id]/page.tsx

```diff
@@ -29,6 +29,7 @@ interface IAssessment {
   level: string;
   name: string;
   questions: number;
+  score: number | undefined;
```

Comment:
Consider using `Null` instead of `Undefined` because param is a defined value rather than an option.
( _Reference link_ : [Null or Undefined](https://www.syncfusion.com/blogs/post/null-vs-undefined-in-javascript) )